// [[Reason: checker.typeToTypeNode deliberately fails on types that originate from node_modules.]] ////

//// [tests/cases/compiler/declarationEmitForGlobalishSpecifierSymlink2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,5 +1,34 @@
 
 
 //// [/p1/index.d.ts]
-export declare const a: import("typescript-fsa").A;
-//# sourceMappingURL=index.d.ts.map
\ No newline at end of file
+export declare const a: invalid;
+//# sourceMappingURL=index.d.ts.map
+/// [Errors] ////
+
+/p1/index.ts(4,14): error TS9010: Variable must have an explicit type annotation with --isolatedDeclarations.
+
+
+==== /cache/typescript-fsa/src/impl.d.ts (0 errors) ====
+    export function getA(): A;
+    export enum A {
+        Val
+    }
+==== /cache/typescript-fsa/index.d.ts (0 errors) ====
+    export * from "./src/impl";
+==== /cache/typescript-fsa/package.json (0 errors) ====
+    {
+        "name": "typescript-fsa",
+        "version": "1.0.0"
+    }
+==== /p1/index.ts (1 errors) ====
+    import * as _whatever from "p2";
+    import { getA } from "typescript-fsa";
+    
+    export const a = getA();
+                 ~
+!!! error TS9010: Variable must have an explicit type annotation with --isolatedDeclarations.
+!!! related TS9027 /p1/index.ts:4:14: Add a type annotation to the variable a.
+==== /p2/index.d.ts (0 errors) ====
+    export const a: import("typescript-fsa").A;
+    
+    
\ No newline at end of file
